.github/workflows-source/release-testing.yml

```diff
@@ -182,10 +193,37 @@ jobs:
           cd ci/src/dependencies/
           $SHELL_WRAPPER python3 job/bazel_rust_ic_scanner_release_job.py
 
-  trigger-qualifier-workflow:
-    name: Qualify
-    needs: [ci-main, bazel-system-test-nightly, bazel-system-test-staging, bazel-system-test-hotfix, dependency-scan-release-cut]
-    uses: dfinity/dre/.github/workflows/qualify.yaml@main
-    secrets: inherit
-    with:
-      version: ${{ github.sha }}
+  setup-guest-os-qualification:
```

Comment:
Doesn't this still need to wait for the other workflows to finish first? I think we want to keep:
```
needs: [ci-main, bazel-system-test-nightly, bazel-system-test-staging, bazel-system-test-hotfix, dependency-scan-release-cut]
```

Reply:
The whole idea is to use this in parallel to speed up the qualification process. The test itself is using the HEAD commit and older versions from mainnet, so we can save a lot of time with this running in parallel

Reply:
would be just good to run after build step if we could.